examples/box_example.cc

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2024 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+//! [complete]
+#include <iostream>
+#include <gz/math/Plane.hh>
+#include <gz/math/Box.hh>
+
+int main(int argc, char**argv)
+{
+
+//! [Create a box]
```

Comment:
this kind of comments `//!` are usefull if you are gogin to write a tutorial too, otherwise you can remove them